icu4c/source/test/intltest/strcase.cpp

```diff
@@ -1370,7 +1372,8 @@ void StringCaseTest::TestCaseMapUTF8WithEdits() {
     // No explicit nor automatic edits.reset(). Edits should be appended.
     length = CaseMap::utf8Fold(U_OMIT_UNCHANGED_TEXT | U_EDITS_NO_RESET |
                                    U_FOLD_CASE_EXCLUDE_SPECIAL_I,
-                               reinterpret_cast<const char*>(u8"IßtanBul"), 1 + 2 + 6, dest, UPRV_LENGTHOF(dest), &edits, errorCode);
+                               reinterpret_cast<const char*>(u8"IßtanBul"), 1 + 2 + 6,
```

Comment:
I suspect this `reinterpret_cast` is no longer needed (such uses were removed in similar tests)

Reply:
On further reflection, I think I was wrong here; the invoked overload is one that has a corresponding `const char*` parameter type.  The other tests where `reinterpret_cast` was removed correspond to cases that use `StringPiece`.